### PR TITLE
Update view-email-security-reports.md

### PR DESCRIPTION
**Pull request by faoquong**
@siosulli

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 1]
```diff
@@ -292,35 +292,35 @@ The following views are available:
 
 - **View data by: Content \> Malware**<sup>1</sup>: The following information is shown for Office 365 ATP organizations:
 
-  - **Anti-malware engine**
-  - **File detonation**
+  - **Anti-malware engine**:Catches of mailcious files in Sharepoint Online, One Drive, and teams by AV
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **Anti-malware engine**: Catches of malicious files in Sharepoint Online, One Drive, and Teams by AV.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 2]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
-  - **ATP-generated URL reputation**<sup>1</sup>
-  - **Advanced phish filter**
-  - **Anti-spoof: DMARC failure**
-  - **Anti-spoof: Intra-org**
-  - **Anti-spoof: external domain**
-  - **Brand impersonation**
-  - **Domain impersonation**<sup>1</sup>
-  - **EOP URL reputation**
-  - **General phish filter**
+  - **ATP-generated URL reputation**<sup>1</sup>: Malicious URL reputation generated from ATP detonations amongst other ATP customers
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **ATP-generated URL reputation**<sup>1</sup>: Malicious URL reputation generated from ATP detonations amongst other ATP customers.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 3]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
+  - **Advanced phish filter** : Phishing signals based on machine learning
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **Advanced phish filter**: Phishing signals based on machine learning.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 4]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
+  - **Anti-spoof: DMARC failure**: DMARC authentication failure on messages
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **Anti-spoof - DMARC failure**: DMARC authentication failure on messages.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 5]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
+  - **Anti-spoof: Intra-org**: sender attempting to spoof the recipient domain
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **Anti-spoof - intra-org**: Sender attempting to spoof the recipient domain.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 6]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
-  - **Anti-spoof: external domain**
-  - **Brand impersonation**
-  - **Domain impersonation**<sup>1</sup>
-  - **EOP URL reputation**
-  - **General phish filter**
+  - **ATP-generated URL reputation**<sup>1</sup>: Malicious URL reputation generated from ATP detonations amongst other ATP customers
+  - **Advanced phish filter** : Phishing signals based on machine learning
+  - **Anti-spoof: DMARC failure**: DMARC authentication failure on messages
+  - **Anti-spoof: Intra-org**: sender attempting to spoof the recipient domain 
+  - **Anti-spoof: external domain**: Sender trying to spoof some other domain.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **Anti-spoof - external domain**: Sender trying to spoof some other domain.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 7]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
+  - **Brand impersonation**: Impersonation of well-known brands based on senders.
+  - **Domain impersonation**<sup>1</sup>: Impersonation of domains that the customer owns or defines.
+  - **EOP URL reputation** : Malicious URL reputation
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **EOP URL reputation**: Malicious URL reputation.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 8]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
+  - **General phish filter**: Phishing signals based on analyst rules. 
   - **Others**
-  - **Phish ZAP**<sup>2</sup>
+  - **Phish ZAP**<sup>2</sup>: Zero hour auto purge phish messages
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **Phish ZAP**<sup>2</sup>: Zero hour auto purge phish messages.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 9]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
+  - **Brand impersonation**: Impersonation of well-known brands based on senders.
+  - **Domain impersonation**<sup>1</sup>: Impersonation of domains that the customer owns or defines.
+  - **EOP URL reputation** : Malicious URL reputation
+  - **General phish filter**: Phishing signals based on analyst rules. 
   - **Others**
-  - **Phish ZAP**<sup>2</sup>
+  - **Phish ZAP**<sup>2</sup>: Zero hour auto purge phish messages 
   - **URL detonation**<sup>1</sup>
-  - **User impersonation**<sup>1</sup>
+  - **User impersonation**<sup>1</sup> :Impersonation of users defined by admin or learned through mailbox intelligence
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **User impersonation**<sup>1</sup>: Impersonation of users defined by admin or learned through mailbox intelligence.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 10]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
 
   ![Detection technology view for phishing email in the Threat protection status report](../../media/threat-protection-status-report-phishing-detection-tech-view.png)
 
 - **Break down by: Detection technology** and **View data by: Email \> Malware**: The following information is shown:
 
-  - **ATP-generated file reputation**<sup>1</sup>
-  - **Anti-malware engine**<sup>1</sup>
-  - **Anti-malware policy file type block**
-  - **File detonation**<sup>1</sup>
+  - **ATP-generated file reputation**<sup>1</sup> :All malicious files reputation generated by ATP detonations
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **ATP-generated file reputation**<sup>1</sup>: All malicious files reputation generated by ATP detonations.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 11]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
+  - **Anti-malware engine**<sup>1</sup> :Detection from anti-malware engines
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **Anti-malware engine**<sup>1</sup>: Detection from anti-malware engines.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 12]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
+  - **Anti-malware policy file type block** :these are email messages filtered out due to the type of malicious file identified in the message
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **Anti-malware policy file type block**: These are email messages filtered out due to the type of malicious file identified in the message.
```

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 13]
```diff
@@ -292,35 +292,35 @@ The following views are available:
...
+  - **File detonation**<sup>1</sup> :File detonation catches by safe attachment
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **File detonation**<sup>1</sup>: File detonation catches by safe attachment.  
```

**Comment by siosulli** [comment]
Thanks @JohanFreelancer9 
 @yogkumgit and @faoquong just to let you know I'm also doing a review on this one!

File: microsoft-365/security/office-365-security/view-email-security-reports.md [hunk 14]
```diff
@@ -292,35 +292,35 @@ The following views are available:
 
 - **View data by: Content \> Malware**<sup>1</sup>: The following information is shown for Office 365 ATP organizations:
 
-  - **Anti-malware engine**
-  - **File detonation**
+  - **Anti-malware engine**:Catches of mailcious files in Sharepoint Online, One Drive, and teams by AV
+  - **File detonation** :Detonation of malicious files in Sharepoint Online, One Drive, and teams by safe attachment.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-email-security-reports.md]:
```suggestion
  - **File detonation**: Detonation of malicious files in Sharepoint Online, One Drive, and teams by safe attachment.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**